appveyor-ubuntu.yml

```diff
@@ -296,7 +296,7 @@ for:
 
     test_script:
       - "pip install -e \".[dev]\""
-      - sh: "pytest -vv tests/integration/delete --json-report --json-report-file=TEST_REPORT-integration-delete.json"
```

Comment:
we could use -n auto as well, as it will scale to number of CPUs available.

Reply:
True, but I didn't want to get throttled in case these are running in an 8 or more core environments.